brainstormforce/ultimate-addons-for-gutenberg
# Resolve an issue of repeated code  get added in style/script tag.

### Description

- Resolve an issue of repeated code  get added in style/script tag

### Screenshots

- NA

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Create a page template with a container
- Add FAQ in header template part and testimonial in the footer template part
- Visit frontend. Search for `-frontend` (make sure file generation is off)

### Checklist:
